setup.py

```diff
@@ -203,6 +203,10 @@ def mark_lhotse_version(version: str) -> None:
     "black==22.3.0",
     "isort==5.10.1",
     "pre-commit>=2.17.0,<=2.19.0",
+    "pandas>2.0.0,<2.3.0",
+    "pyarrow==17.0.0",
+    "fastparquet>2023.10.0,<2024.6.0",
+    "requests==2.31.0",
```

Comment:
Revert these changes, all of those should be optional as they are only needed for this recipe.